src/appengine/libs/issue_management/oss_fuzz_github.py

```diff
@@ -21,6 +21,7 @@
 from clusterfuzz._internal.metrics import logs
 
 GITHUB_PREFIX = 'https://github.com/'
+GITHUB_SUFFIX = '.git'
```

Comment:
nit: this is not necessarily a "GitHub" specific suffix, so let's name this GIT_SUFFIX.